definitions/consented/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -3108,9 +3108,153 @@
     "CaseFieldID": "civilPartnership",
     "PageFieldDisplayOrder": 2,
     "DisplayContext": "OPTIONAL",
-    "PageID": "amendCase",
+    "PageID": 1,
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "17/02/2025",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "CaseEventID": "FR_amendCase",
+    "CaseFieldID": "natureOfApplication2",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY",
+    "PageID": 1,
+    "PageDisplayOrder": 3,
+    "PageColumnNumber": 1,
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/ccdMidEvent",
```

Comment:
Is there a handler for this in finrem-cos?

Reply:
Added a handler for this